common/logger/src/lib.rs

```diff
@@ -263,6 +289,7 @@ macro_rules! sl_level {
     ($level:expr, $entry:expr) => {
         if $crate::struct_log_enabled!($level) {
             let mut entry = $entry;
+            entry.add_category(module_path!().split("::").next().unwrap());
```

Comment:
I was going back and forth on the module, vs category.

This might be good though, the main reason for the category was for something that might be in a different module, but be a different category e.g. security logs.

Reply:
This is the big question I had about all of this. I don't have enough context to know what these "categories" were used for, in most cases it looked like it was just the module name so it just seemed to be extra boilerplate people needed to type. If we can nail down exactly what this is intended to be we can very easily just tack this extra info on to the Schema trait as well